tests/test_sequence.py

```diff
@@ -1,6 +1,7 @@
 import pytest
 
-from vllm.sequence import SamplerOutput, SequenceGroupOutput, SequenceOutput
+from vllm.sequence import (SamplerOutput, SequenceData, SequenceGroupOutput,
+                           SequenceOutput)
```

Comment:
```suggestion
                           SequenceOutput,)
```

and run formatter again this helps reducing the line change on new imports.